### PR TITLE
6517 orapi release

### PR DESCRIPTION
**Pull request by spartanbear**

### CODE REVIEW

File: Kubernetes/BuyingCatalogueOrdering/test-orapi-kubernetes-manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  template:
+    metadata:
+      labels:
+        app: gpitfutures-bc-app-orapi
+    spec:
+      imagePullSecrets:
+      - name: testpipelinevariables
+      containers:
+      - name: gpitfutures-bc-app-orapi
+        image: gpitfuturestestacr.azurecr.io/nhsd/buying-catalogue/orapi
```
Comment (stuart-bass-cgi) [inline Kubernetes/BuyingCatalogueOrdering/test-orapi-kubernetes-manifest.yaml]: image name is inconsistent with dev?

**Review by stuart-bass-bjss — CHANGES_REQUESTED** [review]

File: Kubernetes/BuyingCatalogueOrdering/dev-orapi-kubernetes-manifest.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    matchLabels:
+      app: gpitfutures-bc-app-orapi
+  template:
+    metadata:
+      labels:
+        app: gpitfutures-bc-app-orapi
+    spec:
+      imagePullSecrets:
+      - name: devpipelinevariables
+      containers:
```
Comment (stuart-bass-cgi) [inline Kubernetes/BuyingCatalogueOrdering/dev-orapi-kubernetes-manifest.yml:22]: Should there be liveness / readiness probes?

File: Kubernetes/BuyingCatalogueOrdering/dev-orapi-kubernetes-manifest.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            cpu: 250m
+          limits:
+            cpu: 1
+        livenessProbe:
+          httpGet:
+            path: /health/live
+            port: 80
+        readinessProbe:
+          httpGet:
+            path: /health/live
```
Comment (stuart-bass-cgi) [inline Kubernetes/BuyingCatalogueOrdering/dev-orapi-kubernetes-manifest.yml]: /health/ready

File: Kubernetes/BuyingCatalogueOrdering/test-orapi-kubernetes-manifest.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            cpu: 250m
+          limits:
+            cpu: 1
+        livenessProbe:
+          httpGet:
+            path: /health/live
+            port: 80
+        readinessProbe:
+          httpGet:
+            path: /health/live
```
Comment (stuart-bass-cgi) [inline Kubernetes/BuyingCatalogueOrdering/test-orapi-kubernetes-manifest.yaml]: /health/ready

**Review by stuart-bass-bjss — CHANGES_REQUESTED** [review]

File: Kubernetes/BuyingCatalogueOrdering/test-orapi-kubernetes-manifest.yaml (hunk 4 above)
Comment (spartanbear) [reply Kubernetes/BuyingCatalogueOrdering/test-orapi-kubernetes-manifest.yaml]: Updated readiness Probe for Test

**Review by stuart-bass-bjss — APPROVED** [review]

**Pull request merged**